pkg/ccl/storageccl/export_storage.go

```diff
@@ -268,11 +283,22 @@ func (h *httpStorage) Delete(_ context.Context, basename string) error {
 	return err
 }
 
+func (h *httpStorage) Size(_ context.Context, basename string) (int64, error) {
+	resp, err := h.req("HEAD", basename, nil)
```

Comment:
Some servers might not implement this -- at least it isn't part of our current required API.
We potentially want to add a test where our mock http server errors or otherwise returns an unusable response, just to make sure we keep handling it gracefully.